crates/rattler_installs_packages/src/artifact.rs

```diff
@@ -11,6 +11,8 @@ use std::io::Read;
 use std::str::FromStr;
 use zip::ZipArchive;
 
+/// Trait that represents an artifact type in the PyPi ecosystem.
```

Comment:
Super nit-picky but I think it's PyPI and PyPA (in terms of capitalization).

Reply:
Done